JUnit version 4.12
...E...E....E..E.....E..E....E.E..E.E..E.......E...E.....E..E.
Time: 1.744
There were 15 failures:
1) test02(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test02(NativeArray_ESTest.java:65)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
2) test03(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.ScriptRuntime.toObject(ScriptRuntime.java:978)
	at com.google.javascript.rhino.head.Context.toObject(Context.java:1668)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test03(NativeArray_ESTest.java:84)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
3) test05(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property -319 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test05(NativeArray_ESTest.java:121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
4) test20(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Expecting exception: NullPointerException
	at org.junit.Assert.fail(Assert.java:88)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test20(NativeArray_ESTest.java:306)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
5) test22(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test22(NativeArray_ESTest.java:326)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
6) test01(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property 1498 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test01(NativeArray_ESTest.java:59)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
7) test28(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test28(NativeArray_ESTest.java:383)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
8) test07(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Invalid JavaScript value of type class com.google.javascript.rhino.head.BaseFunction
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test07(NativeArray_ESTest.java:154)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
9) test08(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Expecting exception: NullPointerException
	at org.junit.Assert.fail(Assert.java:88)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test08(NativeArray_ESTest.java:164)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
10) test09(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test09(NativeArray_ESTest.java:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
11) test13(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.AssertionError: Exception was not thrown in com.google.javascript.rhino.head.DefaultErrorReporter but in com.google.javascript.rhino.head.Context.reportRuntimeError(Context.java:947): com.google.javascript.rhino.head.EvaluatorException: Property 2 not found.
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:112)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:49)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test13(NativeArray_ESTest.java:218)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
12) test38(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.ScriptableObject.buildClassCtor(ScriptableObject.java:1261)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue0(LazilyLoadedCtor.java:138)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.access$000(LazilyLoadedCtor.java:51)
	at com.google.javascript.rhino.head.LazilyLoadedCtor$1.run(LazilyLoadedCtor.java:123)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.buildValue(LazilyLoadedCtor.java:119)
	at com.google.javascript.rhino.head.LazilyLoadedCtor.init(LazilyLoadedCtor.java:99)
	at com.google.javascript.rhino.head.ScriptableObject$GetterSlot.getValue(ScriptableObject.java:345)
	at com.google.javascript.rhino.head.ScriptableObject.get(ScriptableObject.java:494)
	at com.google.javascript.rhino.head.IdScriptableObject.get(IdScriptableObject.java:361)
	at com.google.javascript.rhino.head.ImporterTopLevel.get(ImporterTopLevel.java:129)
	at com.google.javascript.rhino.head.ScriptableObject.getProperty(ScriptableObject.java:2213)
	at com.google.javascript.rhino.head.TopLevel.cacheBuiltins(TopLevel.java:114)
	at com.google.javascript.rhino.head.ScriptRuntime.initStandardObjects(ScriptRuntime.java:262)
	at com.google.javascript.rhino.head.Context.initStandardObjects(Context.java:1074)
	at com.google.javascript.rhino.head.ImporterTopLevel.initStandardObjects(ImporterTopLevel.java:106)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:87)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:82)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test38(NativeArray_ESTest.java:479)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
13) test32(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test32(NativeArray_ESTest.java:417)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
14) test17(com.google.javascript.rhino.head.NativeArray_ESTest)
java.lang.NullPointerException
	at com.google.javascript.rhino.head.ImporterTopLevel.initStandardObjects(ImporterTopLevel.java:106)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:87)
	at com.google.javascript.rhino.head.ImporterTopLevel.<init>(ImporterTopLevel.java:82)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test17(NativeArray_ESTest.java:261)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
15) test18(com.google.javascript.rhino.head.NativeArray_ESTest)
org.evosuite.runtime.mock.java.lang.MockRuntimeException: No Context associated with current Thread
	at com.google.javascript.rhino.head.Context.getContext(Context.java:2332)
	at com.google.javascript.rhino.head.NativeArray.getDefaultValue(NativeArray.java:547)
	at com.google.javascript.rhino.head.ScriptRuntime.toNumber(ScriptRuntime.java:410)
	at com.google.javascript.rhino.head.Context.toNumber(Context.java:1622)
	at com.google.javascript.rhino.head.NativeArray_ESTest.test18(NativeArray_ESTest.java:271)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)

FAILURES!!!
Tests run: 47,  Failures: 15

